### PR TITLE
[FTLIS042022] Leticia M.

### PR DESCRIPTION
**Pull request by Pip0kinha**
solve first interaction

### CODE REVIEW

File: javascript/chronometer.js [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60);
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value === 0) {
```
Comment (webLisbonTA) [inline javascript/chronometer.js:25]: You could simplify by omitting the 1st condition as 0 will already be encompassed under the `if(value < 9` condition statement

**Review by webLisbonTA — COMMENTED** [review]
Well done Leticia

**Comment by Pip0kinha** [comment]
solved 2 interation (DOM manipulation)

**Review by webLisbonTA — COMMENTED** [review]
Great job on finishing the lab!

File: javascript/index.js [hunk 2]
```diff
@@ -31,35 +42,61 @@ function printMilliseconds() {
...
-  // ... your code goes here
+  let splitTime = document.createElement('li');
+  splitTime.innerHTML = chronometer.split();
+
+  splitsElement.appendChild(splitTime);
 }
 
 function clearSplits() {
-  // ... your code goes here
+  let li = document.querySelectorAll('li');
```
Comment (webLisbonTA) [inline javascript/index.js:52]: Here, you create the variable but you never use it

**Pull request closed without merging**